.github/workflows/pr-check.yml

```
@@ -14,99 +14,99 @@ jobs:
-    - name: Install dependencies
-      run: npm install --frozen-lockfile
-    
-    - name: Lint
-      run: npm run lint:check -w @ar.io/wayfinder-extension
-    
-    - name: Build core library first
-      run: npm run build -w @ar.io/wayfinder-core
-    
-    - name: Build extension
-      run: npm run build -w @ar.io/wayfinder-extension
+      - uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)